### DISCUSSION TITLE
How to proper configure Zed to recognise all files in the workspace?

### DISCUSSION THREAD

**Discussion by untanglingTheSpaghetti** [zed-industries/zed#15359] [Feature Requests]
Hello!

I am new using Zed, and I was wondering what do I have to do to make Zed to recognise all files in my workspace and define DEFINES.
I am a user of VSC, and I used to use the settings.json file inside .vscode folder to add DEFINES and path to my .h files.
However, in Zed, I do not know how to proper configure it, and as a result, Zed cannot find some of the .h files, that, indeed, they are in the workspace.

Thank you for your help!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
